flagsmith-engine/segments/models.ts

```diff
@@ -1,4 +1,4 @@
-import semver from 'semver';
+import * as semver from 'semver';
```

Comment:
Is this just convention?

Reply:
This is needed after removing https://www.typescriptlang.org/tsconfig/#esModuleInterop, which is more compliant with ES modules. AFAIK there's no actual impact on the generated code or customer experience.